easybuild/easyconfigs/g/GetOrganelle/GetOrganelle-1.7.5.3-foss-2021b.eb

```
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'GetOrganelle'
+version = '1.7.5.3'
+
+homepage = 'https://github.com/Kinggerm/GetOrganelle'
+description = """This toolkit assemblies organelle genome from genomic skimming data."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/Kinggerm/GetOrganelle/archive/']
+sources = ['v%(version)s.tar.gz']
```

https://github.com/Kinggerm/GetOrganelle/commits/master
@boegel: looks like they've indeed re-released this version, and removed the `v` from the name of the tarball.

```suggestion
sources = ['%(version)s.tar.gz']
```

---

Indeed, and it seems like that's the only thing that has changed, the checksum is exactly the same as it was for `v1.7.5.3.tar.gz`, so I guess they just fixed the tag name they were using (from `v1.7.5.3` to `1.7.5.3`), to be in line with previous releases (cfr. https://github.com/Kinggerm/GetOrganelle/tags)